config/base/services/authn-webhook/kustomization.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- authn-webhook.yaml
+- authn-webhook-service.yaml
+images:
+- name: auth-provider-zitadel
+  newName: auth-provider-zitadel
+  newTag: dev
```

We can leave this as `latest` by default.
```suggestion
- name: ghcr.io/datum-cloud/auth-provider-zitadel
  newTag: latest
```

---

Done!